github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -45,13 +45,18 @@ presubmits:
           value: "kind-1.19-sriov"
         - name: "KUBEVIRT_NUM_NODES"
           value: "3"
+        - name: "RUN_KUBEVIRT_CONFORMANCE"
+          value: "true"
+        - name: "SONOBUOY_EXTRA_ARGS"
+          value: "--plugin-env kubevirt-conformance.E2E_FOCUS=SRIOV"
         command:
         - "/usr/local/bin/runner.sh"
         - "/bin/bash"
-        - "-c"
-        - >
-            trap "echo teardown && make cluster-down" EXIT ERR SIGINT SIGTERM &&
-            make cluster-up
+        - "-ce"
+        - |
+          trap "echo teardown && make cluster-down" EXIT SIGINT
+          make cluster-up
+          ./cluster-up/cluster/kind/check-cluster-up.sh
```

Comment:
Where is this script defined? I don't see it in https://github.com/kubevirt/kubevirtci/tree/main/cluster-up/cluster/kind

Reply:
Oh, I see it here https://github.com/kubevirt/kubevirtci/pull/645/files sorry for the noise

Reply:
This build https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/645/check-up-kind-1.19-sriov/1425033590051180544 is running with the changes in this PR plus the changes in https://github.com/kubevirt/kubevirtci/pull/645

Reply:
It failed with `error: timed out waiting for the condition on pods/kube-controller-manager-sriov-control-plane` here https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/645/check-up-kind-1.19-sriov/1425033590051180544#1:build-log.txt%3A2876 

@ormergi can you pls take a look?

Reply:
@fedepaol I pushed new change for kubevirt/kubevirt#645 lets run it again

Reply:
Sure, running here https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/645/check-up-kind-1.19-sriov/1425042102244347904

Reply:
@fedepaol I see tests are failing due to not passing `container-tag`, kubevirt/kubvirt#6199 addresses this problem.